docs/distributions/gentoo/installation.md

```diff
@@ -0,0 +1,54 @@
+# Installing Gentoo Linux on a T2 Mac
+
+## Hardware Requirements
+* USB-C to USB-A adapter
+* USB Ethernet Adapter/USB Wi-Fi Adapter
+	* This is not required, but if you're using a normal Gentoo Minimal Installation ISO, then it is required.
+	* If you end up not using this, then you are required to use another distro's ISO in order to connect to the internet via Wi-Fi.
+* USB Flash Drive
+
+## Install Procedure
+1. Partition your SSD
+	1. Open the Bootcamp installer and follow along until it requests for you to input a Windows ISO. This should clear space for a Linux partition because of removed APFS snapshots.
+	2. Now open up Disk Utility. Make a partition with any format. The amount of space you allocate for this Linux partition will be final, so choose wisely.
+2. Download a Gentoo ISO and flash it to your USB Flash Drive via [Balena Etcher](https://www.balena.io/etcher/) (you can also use dd)
+3. Ensure that Secure Boot is disabled
+	1. Follow the instructions from [this article](https://support.apple.com/en-us/HT208198)
+	2. Once in the Startup Security Utility, set secure boot to no security and enable external boot.
+4. Boot into the Live USB enviornment
+	1. Plug in the USB Flash Drive into your computer
+	2. Startup while holding the Option key, this will put you in the macOS Startup Manager
+	3. Select the orange EFI Boot option and press enter to boot into it. (If you're using a Ubuntu Live enviornment, then make sure to select the orange EFI Boot option to the far right)
```

Comment:
Maybe `far right` is a bit unfortionate wording here, how about `all the way to the right`